config/cluster.yaml

```diff
@@ -0,0 +1,14 @@
+##  cluster settings
+# id: <id of the cluster to be created>
```

Comment:
I would show the documentation above each entry instead of at the top
```yaml
# Id of the cluster to be created
id: spark_cluster

# VM Size, see available options here: https://azure.microsoft.com/en-us/pricing/details/virtual-machines/linux
vm_size: standard_a2
```